src/schemas/auth.ts

```diff
@@ -0,0 +1,8 @@
+import { Static, Type } from "@sinclair/typebox";
+
+export const CredentialsSchema = Type.Object({
+  username: Type.String(),
+  password: Type.String()
+});
+
+export interface IAuth extends Static<typeof CredentialsSchema> {}
```

Comment:
Most of the time, I will override the types when using TypeBox.
So, the schema will match what the expected types.

```suggestion

export type CredentialsSchema = Static<typeof CredentialsSchema>
export const CredentialsSchema = Type.Object({
  username: Type.String(),
  password: Type.String()
});
```

Reply:
The problem is that `@typescript-eslint/no-redeclare` is currently enforced

Reply:
I prefer to have confirmation from other contributors before proceeding, but I have nothing against it.